xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
 ${Search Control(Start) - shows search button, options and textbox}
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+<input class="query-textbox" id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
 <script>
 if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
-<input type=submit value="Search">
+<input class="query-search-button" type=submit value="Search">
 ${Todo: Replace radios with search buttons}
 $if{,$include{inc/anyalldropbox},
-<fieldset>$include{inc/anyallradio}</fieldset>}
+<fieldset class="query-options">$include{inc/anyallradio}</fieldset>}
+</div>
 ${Search Control(End)}
 ${Suggestions(Start) - shows suggested terms to improve query}
 $if{$opt{topterms},
- <div title="Suggested terms to add to your query"
+ <div class="query-suggested-container" title="Suggested terms to add to your query"
```

Comment:
It's utterly related to the form: it's a series of checkboxes for adding terms to add to the query (as the `div`'s title says).